data/sql/updates/pending_db_world/rev_1591992052509403400.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1591992052509403400');
+
+DELETE FROM `playercreateinfo_item` WHERE `itemid` = 40582 AND `amount` = -1;
+
+ALTER TABLE `playercreateinfo_item`
+	CHANGE `amount` `amount` SMALLINT UNSIGNED NOT NULL DEFAULT 1,
+	ADD `Note` VARCHAR(255) AFTER `amount`;
```

Comment:
what is that `note` column about? @Kitzunu i feel like it's a mistake

Reply:
So you dont have to google what the race and item are. So you can add a comment like "Warrior - Shadowedge"

Reply:
call it "comment" then? like in other tables

and yes i can see it being useful, it won't take much space :D

Reply:
I call it note because other playercreateinfo call it that

Reply:
hmm it's true on playercreateinfo_spell and spell_custom even though it's shit

dunno, mixed feelings

Reply:
better do it this way and then refactor in the future imo

Reply:
in fact the note is not used by the core so we could just rename to comment in this pr and also for the 2 other tables xD